plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -255,6 +252,138 @@ p. With this definition we can use expressions like @t.a.x@, see the image below
 p=. Content assist in data structure traversal
 
 
+h2(#cdom_pointers). Pointers
+
+Pointers are a core feature of the C programming language. YAKINDU Statechart Tools' Deep C Integration is making C pointers available to you in your statecharts. In particular, you can
+* declare typed pointer variables,
+* assign a pointer to a pointer variable of the same type,
+* retrieve the pointer pointing to a given variable,
+* pass pointers as parameters to functions, and
+* receive a pointer as a functions return value.
+
+h3(#cdom_declaring-pointer-variables). Declaring pointer variables
+
+Pointer variables are declared in a statechart's definition section as shown in the following example:
+
+bc. var n: int32_t
+var pInt: pointer<int32_t>
+var ppInt: pointer<pointer<int32_t> >
+var pTriangle: pointer<Triangle>;
+
+
+The declarations above declare
+* _n_ as a (non-pointer) variable of type _int32_t_,
+* _pInt_ as a pointer variable that is pointing to a variable of type _int32_t_,
+* _ppInt_ as a pointer that is pointing to a pointer that is pointing to a variable of type _int32_t_, and
+* _pTriangle_ as a pointer to a variable of the self-defined type _Triangle_.
+
+bq.. *Please note:*
+
+When closing the type specification in a pointer declaration with angle brackets, e.&nbsp;g. @pointer<pointer<int32_t> >@, the @>@ characters must be separated from each other by one or more white space characters. Writing e.&nbsp;g. @pointer<pointer<int32_t>>@ would result in an error.
+
```

Comment:
Maybe point out that this is only the current state and will be resolved in a later version.